src/Attachment/File.php

```diff
@@ -54,19 +60,21 @@ class File
      * @param string|null  $disk
      * @param string|null  $group
      */
-    public function __construct(UploadedFile $file, string $disk = null, string $group = null)
+    public function __construct(UploadedFile $file, string $disk = null, string $path = null, string $group = null)
     {
         abort_if($file->getSize() === false, 415, 'File failed to load.');
 
         $this->file = $file;
 
+        $this->replicate = true ;
+        $this->path = $path;
```

Comment:
I don't really understand why we are doing these properties in the current context

Reply:
those are irrelevant to the current context, there has been a mix, sorry for this.